src/Microsoft.Diagnostics.EventFlow.Inputs.MicrosoftLogging/Logger/EventFlowLogger.cs

```diff
@@ -61,17 +64,22 @@ public void Log<TState>(Extensions.Logging.LogLevel logLevel, EventId eventId, T
                         }
 
                         //last KV is the whole 'scope' message, we will add it formatted
-                        AddPayloadProperty(properties, "Scope", formattedState.ToString());
+                        scopePropertyValues.Add(formattedState.ToString());
                     }
                     else
                     {
-                        AddPayloadProperty(properties, "Scope", scope.State);
+                        scopePropertyValues.Add(scope.State.ToString());
                     }
                 }
 
                 scope = scope.Parent;
             }
 
+            if (scopePropertyValues.Count > 0)
+            {
+                AddPayloadProperty(properties, "Scope", string.Join(" > ", scopePropertyValues.AsEnumerable().Reverse()));
```

Comment:
1. Could we use a stack, instead of a list, to avoid the Reverse() call?
2. I wonder if a better separator would be \r\n? I am afraid things might get a little confusing if any of the scope strings contain ">"